README.md

```diff
@@ -69,11 +69,15 @@ use the [transform-decorators-legacy] plugin. See this [post] for detailed instr
 
 ## Decorators
 
-### `@controller(basePath)`
+### `@controller(basePath, optionsKey = 'config')`
```

Comment:
I don't think we need to do this, I'm for doing a major change and bumping it to v1 (we could do v0.5, but v1 would be more curious to existing users). So defaulting to what v17 Hapi support and up for v1.